cli/command_cache_set.go

```diff
@@ -4,21 +4,42 @@ import (
 	"context"
 	"time"
 
+	"github.com/alecthomas/kingpin/v2"
 	"github.com/pkg/errors"
 
 	"github.com/kopia/kopia/internal/units"
 	"github.com/kopia/kopia/repo"
 	"github.com/kopia/kopia/repo/content"
 )
 
+type cacheSizeFlags struct {
+	contentCacheSizeMB          int64
+	maxMetadataCacheSizeMB      int64
+	contentCacheSizeLimitMB     int64
+	maxMetadataCacheSizeLimitMB int64
+	maxListCacheDuration        time.Duration
```

Comment:
nit; Might be better to rename `maxMetadataCacheSizeLimitMB` -> `metadataCacheSizeLimitMB` and `maxMetadataCacheSizeMB ` -> `metadataCacheSizeMB ` to be consistent with the other names.

Reply:
done